### DISCUSSION TITLE
Couldn't set association for project. Check the file is writeable

### DISCUSSION THREAD

**Discussion by FrankNote** [microsoft/AirSim#3571] [Support Q&A]
couldn't generate vs project files,what can i do?

**Comment by rajat2004** [comment]
Have you tried the possible solutions found by googling this error? These might be useful - https://answers.unrealengine.com/questions/164472/couldnt-set-association-for-project-check-the-file.html?sort=newest, https://answers.unrealengine.com/questions/382018/view.html
